### PR TITLE
requested reviewers are not @mentioned in chat

### PR DESCRIPTION
**Pull request by goern**
Signed-off-by: Christoph Görn <goern@redhat.com>

### CODE REVIEW

File: aicoe/sesheta/utils.py [hunk 1]
```diff
@@ -38,6 +38,55 @@
...
+    "dfeddema": "Diane Feddema",
+    "durandom": "Marcel Hild",
+    "cermakm": "Marek Cermak",
+    "hemajv": "Hema Veeradhi",
+    "zmhassan": "Zak Hassan",
+    "humairak": "Humair Khan",
+    "sesheta": "Thoth Bot",
+}
+
+REALNAME_HANGOUTS_MAP = {
```
Comment (harshad16) [inline aicoe/sesheta/utils.py]: I don't if we should add **user_ids** in the code, may using the api logic to get the user_id as each query sounds compute, as we don't emphasis on the compute time. 
May be we should do a lookup of member from channel.
Logic in older code: 
```
response = chat.spaces().members().list(parent=<channel_space>)
result = response.execute()
for member in result["memberships"]:
    if member["member"].get("displayName") == <user>:
        return member["member"].get("name")
```

Comment (goern) [reply aicoe/sesheta/utils.py]: :) feel free to approve this and send a patch :package:

**Comment by goern** [comment]
hmm, zuul is not merging it?!

**Comment by harshad16** [comment]
> hmm, zuul is not merging it?!

approvers need to be someone with write access

**Pull request merged**